Assets/Scripts/Cards/CardInShop.cs

```diff
@@ -48,4 +53,9 @@ public void OnPointerExit(PointerEventData eventData)
         Debug.Log("Receive event");
         Destroy(this.gameObject);
     }
+
+    public void sold() {
+        soldOutSign.SetActive(true);
+        cardDisplay.gameObject.SetActive(false);
```

Comment:
Did a test, since the onclick handler is in this class, even after buying the card you can go to buy it again. Will add it to the trello under bug fix.